src/validation/rooms.validation.js

```diff
@@ -0,0 +1,52 @@
+import { check, validationResult } from 'express-validator';
+
+// this transforms the default error message
+const customMessage = validationResult.withDefaults({
+  formatter: (error) => (
+    {
+      field: error.param,
+      message: error.msg
+    }
+  )
+});
+
+const validateRooms = [
+  check('name', 'Kindly Provide a name for your room')
```

Comment:
`name` here should have letters too (like "Room 5", as saying just "5" won't make much sense.

Reply:
About this, we can't really influence how people will want to name the rooms on their facilities. But I will go ahead and Implement this if you say so. Thanks for the feedback